CHANGELOG.md

```diff
@@ -49,8 +49,8 @@
 * [ENHANCEMENT] Added new Traces api V2[#3912](https://github.com/grafana/tempo/pull/3912) (@javiermolinar)
 * [ENHANCEMENT] Update to the latest dskit [#3915](https://github.com/grafana/tempo/pull/3915) (@andreasgerstmayr)
 * [ENHANCEMENT] Reduce allocs building queriers sharded requests [#3932](https://github.com/grafana/tempo/pull/3932) (@javiermolinar)
-
 * [ENHANCEMENT] Implement polling tenants concurrently [#3647](https://github.com/grafana/tempo/pull/3647) (@zalegrala)
+* [ENHANCEMENT] Fix broken links in the documentation [#3956](https://github.com/grafana/tempo/pull/3956) (@javiermolinar)
```

Comment:
we normally don't do changelog entries for the docs, but i'm not opposed to it. if we do start this i think we should mark them separately tho.

`[DOC] Fix broken lines ... `

Reply:
ah didn't know, I can remove it